src/libsync/logger.cpp

```diff
@@ -146,10 +147,12 @@ void Logger::doLog(QtMsgType type, const QMessageLogContext &ctx, const QString
     {
         QMutexLocker lock(&_mutex);
 
-        if (_logFile.size() >= MaxLogSizeBytes) {
+        if (linesCounter >= MaxLogSizeBytes) {
```

Comment:
@mgallien since this `MaxLogSizeBytes` is no longer used to compare against size, does it make sense to keep its name like this? feels confusing

Reply:
done